Image_Viewer/menu.py

```diff
@@ -18,10 +20,10 @@ def create_menu(self):
 
         theme_menu = Menu(menubar, tearoff=0)
         theme_menu.add_command(
-            label="Dark", command=lambda: self.set_theme("#1D1D1D", "#F0F0F0", "#1C1C1C"), underline=0
+            label="Dark", command=lambda:self.change_theme("Dark"), underline=0
```

Comment:
Here's an idea around the arguments of `change_theme(...)` which isn't what I'd consider to be a blocker to merging, but I'll share it with you anyway:

`"Dark"` and `"Light"` could be considered as "magic strings", meaning it's a string that is defined specifically in the application code. These strings have to match the key in the JSON file.

We may need to do `change_theme("Dark")` elsewhere (for example, there could be another route the user could take to change their theme in the future). If we changed the key in the JSON file, we would have to do a find and replace to change any occurrences of that "Dark" string. There could also be problems if a typo is made in the string that is passed into `change_theme()`, as it would mean that the key isn't found.

What we could do is create a class that contains constant strings for the themes, it could look something like this (the caps are convention):
![image](https://user-images.githubusercontent.com/54030147/197830896-bb80571d-124f-4bc4-b312-34309bd64986.png)

This means that if a change needed to be made to a string, that can be done in one place, rather than trying to find all occurrences of that string.

To use that constant string, you'd do this:
![image](https://user-images.githubusercontent.com/54030147/197831466-4c7d1213-07ef-4bae-a174-e45b20265eb6.png)
(Note: I left the label deliberately in that example as that's a UI string, which is being accounted for in #28, so you wouldn't need to worry about that, and I wouldn't put UI strings in a constant strings file as that would make things a bit challenging when it comes to adding new languages).

I'll leave it up to you as to whether you make this change (or what other ways magic strings can be handled!), I personally think it's nice to have and would set a good example to any future collaborators!

Reply:
Good thinking! I'll let you implement whichever you think would be best :)